### PR TITLE
add PROJECT.md file

### PR DESCRIPTION
**Pull request by jorydotcom**
Per bootstrap meeting discussion on 3/25, adds list of current projects with their stages.

### CODE REVIEW

File: PROJECTS.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# OpenJS Foundation Projects
+
+The projects below are full OpenJS Foundation members at the stage level indicated in the heading. Projects may move to new stages upon annual review or by request. 
+
+For more information on the stage definitions and how projects move between stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.
```
Comment (Trott) [inline PROJECTS.md]: Optional text simplification:

```suggestion
For information on the stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.
```

File: PROJECTS.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Impact Projects
+
+* [Appium](http://appium.io/)
+* [Dojo](https://dojo.io/)
+* [jQuery](https://jquery.com/)
+* [Node.js](https://nodejs.dev/)
```
Comment (Trott) [inline PROJECTS.md]: The link should be https://nodejs.org/, no?

Comment (Trott) [reply PROJECTS.md]:
```suggestion
* [Node.js](https://nodejs.org/)
```

File: PROJECTS.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# OpenJS Foundation Projects
+
+The projects below are full OpenJS Foundation members at the stage level indicated in the heading. Projects may move to new stages upon annual review or by request.
```
Comment (Trott) [inline PROJECTS.md]: Optional simplification: Split long-ish sentence into two short and direct sentences:

```suggestion
The projects below are full OpenJS Foundation members. They are at the stage level indicated in the headings. Projects may move to new stages upon annual review or by request. 
```

File: PROJECTS.md (hunk 2 above)
Comment (jorydotcom) [reply PROJECTS.md]: Whatever ya'll prefer! I wondered if you'd like .dev site because its new and shiny? No matter to me!

Comment (Trott) [reply PROJECTS.md]: The .dev site is a tutorial. The .org site is where you go to download Node.js, read the reference documentation, etc. I would definitely prefer the .org site.

File: PROJECTS.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* [MochaJS](https://mochajs.org/)
+* [Node-RED](https://nodered.org/)
+* [WebdriverIO](https://webdriver.io/)
+* [webhint](https://webhint.io/)
+
+## At Large Projects
+
+* [ESLint](https://eslint.org/)
+* [Esprima](http://esprima.org/)
+* [Express](https://expressjs.com/)
```
Comment (mhdawson) [inline PROJECTS.md]: Still trying to get feedback from Doug on this front.

**Comment by joesepi** [comment]
I agree with @Trott but otherwise LGTM

File: PROJECTS.md (hunk 4 above)
Comment (jorydotcom) [reply PROJECTS.md]: per conversation in meeting today, moving Express to 'TBD'

**Comment by jorydotcom** [comment]
Thanks @Trott @mhdawson @joesepi <3

**Comment by mhdawson** [comment]
Oops, see it already landed :)

**Pull request merged**